lib/widgets/poll.dart

```diff
@@ -68,15 +68,14 @@ class _PollWidgetState extends State<PollWidget> {
           store.users[userId]?.fullName ?? zulipLocalizations.unknownUserName)
         .join(', ');
 
-      return Padding(
-        padding: const EdgeInsets.only(bottom: 5),
-        child: Row(
-          spacing: 5,
-          crossAxisAlignment: CrossAxisAlignment.baseline,
-          textBaseline: localizedTextBaseline(context),
-          children: [
-            ConstrainedBox(
-              constraints: const BoxConstraints(minWidth: 25).tighten(height: 25),
+      return Row(
```

Comment:
```
poll [nfc]: Move vote count paddings to a box
```

> This does not visually change the paddings.

This doesn't seem to be true. Here's before/after this change, with message https://chat.zulip.org#narrow/stream/7-test-here/topic/new.20poll.20test/near/1956979 :

| Before | After |
| --- | --- |
| <img width="559" alt="image" src="https://github.com/user-attachments/assets/0d7bd6d8-f7ac-4aaf-b0be-eff246266a97"> | <img width="559" alt="image" src="https://github.com/user-attachments/assets/ab890e47-d01b-41a7-802c-ed75d28b7e8b"> |

When I open the two images in different browser tabs and switch between them, I see a layout change. Do you reproduce?

Reply:
Hmm, I can see that in the screenshots, but not on my Android device.

Reply:
When the `bottom: 5` padding wraps the whole row (before this commit), instead of just the vote-count rectangle, there will be 5px whitespace below the text (option name + voters' names) even when the text takes more vertical space than the vote-count rectangle.

I think you can reproduce if you make an option with a name so long that it takes two or three lines on your device.